spyder/widgets/collectionseditor.py

```diff
@@ -1383,21 +1368,36 @@ def oedit(self, key):
             oedit)
         oedit(data[key])
 
+    def __get_pyplot(self):
+        """Return default plotting library `pyplot` package if available.
+        (default plotting library is an option defined in Variable Explorer 
+        plugin scope, e.g. "maplotlib" or "guiqwt")"""
+        libname = self.get_conf('plotlib', section='variable_explorer')
+        try:
+            return __import__(libname + '.pyplot',
+                              globals(), locals(), [], 0).pyplot
+        except ImportError:
```

Comment:
```suggestion
        except Exception:
```

Libraries can fail to import for all kinds of reasons, so better to be safe here.